settings/src/test/java/fr/pilato/elasticsearch/crawler/fs/settings/FsSettingsParserTest.java

```diff
@@ -77,7 +77,11 @@
             .setProtocol("SSH")
             .setPemPath("/path/to/pemfile")
             .build();
-    private static final ServerUrl REST_FULL = new ServerUrl("http://127.0.0.1:8080/fscrawler");
+    private static final Rest REST_EMPTY = Rest.builder().build();
```

Comment:
We are not using `REST_EMPTY`. Could you remove that or add a test that uses it?